### PR TITLE
Content updates 

### LINKED ISSUE #339: Changes to "Your account" Flow
@pcraig3 

**Your account screen**

Please change “Your account” to “Manage your account” (both for page heading and header in top nav bar)

Mobile phone number  - please replace "N/A" with “Not using”

Security codes - please replace “None generated” with “None used yet” 

**Security codes screen** (aka Offline 2FA logged in screen)

Please change “Back to your account”  to  “Back to Manage your account” 

Please change the French versions of the 2 screens above as shown in the mock-ups: https://www.figma.com/file/znPqLTQQDcUkaNjdDcShiz/Exposure-Notification---Health-Portal?node-id=4361%3A0

### LINKED ISSUE #340: Changes to "Change your password" flow
@pcraig3 

Please change as follows in both English and French
All the hint text is on the screen, but there should be just one line: “Your password must be at least 12 characters and hard to guess.” 
The other hint text is only meant to show if the user makes that particular error.

Please replace 
"Confirm your password
Enter the same password as above."
 with “Enter your password again”   and in French with "Saisissez de nouveau votre mot de passe"

### LINKED ISSUE #343: Changes to "Onboarding /Account creation" flow
@pcraig3 

**Create account screen** aka Account Creation

Please change as follows 
All the hint text is given but there should be one line: “Your password must be at least 12 characters and hard to guess.” 
The other hint text is only meant to show if the user makes that particular error.

Please replace
Confirm your password
Enter the same password as above.
 with “Enter your password again.” and in the French as this mockup "Saisissez de nouveau votre mot de passe."

Please change “Create account” to “Next”

**Enter mobile phone number screen**
Under the heading, please remove “Mobile phone number”

Please change the French version to follow this mock-up https://www.figma.com/file/znPqLTQQDcUkaNjdDcShiz/Exposure-Notification---Health-Portal?node-id=4571%3A29106

**Enter your security code screen -** aka Two-Factor (no mobile)
Please change” You kept these codes on paper.” To “You kept these codes somewhere safe”
Please remove “Security code” over box

Please change French version to this mock-up https://www.figma.com/file/znPqLTQQDcUkaNjdDcShiz/Exposure-Notification---Health-Portal?node-id=4571%3A28908

**Enter one of your security codes screen** aka Two-Factor mobile + backup codes
Please change to “We’ve texted you a code and you also kept codes somewhere safe.”
Please remove “Security code” over box

Please change French version to this mock-up https://www.figma.com/file/znPqLTQQDcUkaNjdDcShiz/Exposure-Notification---Health-Portal?node-id=4740%3A12

**No security code screen** aka Resend Two-Factor (no mobile)
Please change to Ask your administrator for a code. You can email them at  admin@email.com.  
And in French please change to: Demandez à votre administrateur de vous donner un code. Vous pouvez leur envoyer un courriel à admin@email.com.

**Request for security codes email**
Please replace “To generate security codes” with “To give them a code:”
Please replace  “Portal” with “portal”
Please replace “Edit” with “Manage”
On line Go to ‘Security codes’ row. select ‘Get security code’. Please remove “select ‘Get security code”
On next line please remove space between ‘ and Get
In the line “Read. . . “ please remove the “s” at the end of codes.
Please replace “Placeholder for French” with the French.

Please reverse the order for emails to Quebec.

### PR DESCRIPTION
**Pull request by pcraig3**
# Summary

👉 **The link is here: https://covid-alert-content-upd-cee4uu.herokuapp.com/**

This PR: 
- resolves #343 
- resolves #340 
- resolves #339

Only content updates here, no logic changes at all.

## Test instructions 

Look through the issues and see if the content has been updated in English and French.

### CODE REVIEW

File: profiles/forms.py [hunk 1]
```diff
@@ -348,12 +351,19 @@ class SignupForm(HealthcareBaseForm, UserCreationForm, forms.ModelForm):
...
 
-    # overwrite the password1 validation message in the __init__ method
-    # password1 is inherited from the UserCreationFrom base class: https://github.com/django/django/blob/302caa40e4caab7d95ef7d0a88a90f935039ab09/django/contrib/auth/forms.py#L90
+    # password1 overwrites the one in the UserCreationFrom base class: https://github.com/django/django/blob/302caa40e4caab7d95ef7d0a88a90f935039ab09/django/contrib/auth/forms.py#L90
+    password1 = forms.CharField(
+        label=_("Password"),
+        strip=False,
+        widget=forms.PasswordInput(),
+        help_text=_(
+            "Choose a password that is hard to guess, with at least 12 characters including 1 letter.\n The password cannot include your name, email or words related to Covid or the portal."
```
Comment (bryan-robitaille) [inline profiles/forms.py:360]: Wondering if 'Covid' should be 'COVID'

File: profiles/forms.py [hunk 2]
```diff
@@ -303,19 +305,20 @@ def send_mail(
 class HealthcarePasswordResetConfirm(HealthcareBaseForm, SetPasswordForm):
     new_password1 = forms.CharField(
         label=_("New password"),
-        widget=forms.PasswordInput(),
         strip=False,
-        help_text=password_validation.password_validators_help_text_html(),
+        widget=forms.PasswordInput(),
+        help_text=_(
+            "Choose a password that is hard to guess, with at least 12 characters including 1 letter.\n The password cannot include your name, email or words related to Covid or the portal."
```
Comment (bryan-robitaille) [inline profiles/forms.py:311]: Wondering if 'Covid' here should be 'COVID'.
Noticed in the content request it was not capitalized but in the rest of the app it is.

File: profiles/forms.py [hunk 3]
```diff
@@ -199,12 +199,14 @@ class HealthcarePasswordEditForm(HealthcareBaseEditForm):
         label=_("Password"),
         strip=False,
         widget=forms.PasswordInput(),
-        help_text=password_validation.password_validators_help_text_html(),
+        help_text=_(
+            "Choose a password that is hard to guess, with at least 12 characters including 1 letter.\n The password cannot include your name, email or words related to Covid or the portal."
```
Comment (bryan-robitaille) [inline profiles/forms.py:203]: Same as others... should 'Covid' be 'COVID'

**Review by bryan-robitaille — APPROVED** [review]
Approving but leaving a question around whether 'Covid' in the hint text should be 'COVID'.

File: profiles/forms.py (hunk 1 above)
Comment (pcraig3) [reply profiles/forms.py:360]: You're right! Making the change.

**Pull request merged**